concepticondata/conceptlists/Koelle-1854-280.tsv

```diff
@@ -0,0 +1,322 @@
+Koelle-1854-280-221c	221c	(of things)	New	qlt	1231	NEW
+Koelle-1854-280-222	222	Sick	Sick	N	1847	SICK
+Koelle-1854-280-223	223	Well	Well	N	1364	HEALTHY
+Koelle-1854-280-224	224	Hot	Hot	qlt	1286	HOT
+Koelle-1854-280-225	225	Cold	Cold	qlt	1287	COLD
+Koelle-1854-280-226	226	Wet	Wet	qlt	1726	WET
+Koelle-1854-280-227	227	Dry	Dry	qlt	1398	DRY
+Koelle-1854-280-228	228	Greedy	Greedy	qlt	2018	GREEDY
+Koelle-1854-280-229	229	Stupid	Stupid	N	1518	STUPID
+Koelle-1854-280-230	230	Rich	Rich	N	712	RICH
+Koelle-1854-280-231	231	Poor	Poor	N	1674	POOR
+Koelle-1854-280-232	232	Straight	Straight	N	1404	STRAIGHT
+Koelle-1854-280-233	233	Crooked (Bent)	Crooked (Bent)//Crooked (bent)	qlt	297	CROOKED
+Koelle-1854-280-233a	233a	(of men)		qlt	565	OLD (OF MAN)
+Koelle-1854-280-234	234	I go	I go	V	695	GO
+Koelle-1854-280-235	235	I come	I come	V	1446	COME
+Koelle-1854-280-236	236	I run	I run	V	1519	RUN
+Koelle-1854-280-237	237	I stop	I stop	V	2880	HALT (STOP)
+Koelle-1854-280-238	238	I sit down	I sit down	V	1649	SIT DOWN
+Koelle-1854-280-239	239	I lie down	I lie down	V	215	LIE DOWN
+Koelle-1854-280-240	240	I breathe	I breathe	V	1407	BREATHE
+Koelle-1854-280-241	241	I cough		V	879	COUGH
+Koelle-1854-280-241a	241a	cough	cough	V	879	COUGH
+Koelle-1854-280-242	242	I sneeze	I sneeze	V	1621	SNEEZE
+Koelle-1854-280-243	243	I snore	I snore	V	1983	SNORE
+Koelle-1854-280-244	244	I laugh	I laugh	V	1355	LAUGH
+Koelle-1854-280-245	245	I weep	I weep	V	1839	CRY
+Koelle-1854-280-246	246	I kneel	I kneel	V	66	KNEEL
+Koelle-1854-280-247	247	I dream	I dream	V	1920	DREAM (SOMETHING)
+Koelle-1854-280-248	248	I sleep	I sleep	V	1585	SLEEP
+Koelle-1854-280-249	249	I die	I die	V	1494	DIE
+Koelle-1854-280-250	250	I fall	I fall	V	1280	FALL
+Koelle-1854-280-251	251	I rise	I rise	V	1512	RISE (ONESELF)
+Koelle-1854-280-252	252	I speak	I speak	V	1623	SPEAK
+Koelle-1854-280-253	253	I hear	I hear	V	1408	HEAR
+Koelle-1854-280-254	254	I beg	I beg	V	1828	ASK (REQUEST)
+Koelle-1854-280-255	255	I bathe (wash myself)	I bathe (wash myself)	V	138	BATHE
+Koelle-1854-280-256	256	I see	I see	V	1409	SEE
+Koelle-1854-280-257	257	I take	I take	V	1749	TAKE
+Koelle-1854-280-258	258	I buy	I buy	V	1869	BUY
+Koelle-1854-280-259	259	I sell	I sell	V	1571	SELL
+Koelle-1854-280-260	260	I love thee		V	923	LOVE
+Koelle-1854-280-261	261	I give thee	I give thee	V	1447	GIVE
+Koelle-1854-280-262	262	I eat rice (yam)	I eat rice	V	1336	EAT
+Koelle-1854-280-262b	262b	I eat yam	I eat yam	V	1336	EAT
+Koelle-1854-280-262c	262c	I eat yams	I eat yams	V	1336	EAT
+Koelle-1854-280-262d	262d	I eat corn	I eat corn	V	1336	EAT
+Koelle-1854-280-263	263	I drink water	I drink water	V	1401	DRINK
+Koelle-1854-280-264	264	I cook meat	I cook meat	V	1100	COOK (SOMETHING)
+Koelle-1854-280-265	265	I kill a fowl	I kill a fowl	V	1417	KILL
+Koelle-1854-280-266	266	I cut a tree	I cut a tree	V	1432	CUT
+Koelle-1854-280-267	267	I flog a child	I flog a child	V		
+Koelle-1854-280-268	268	I catch a fish	I catch a fish	V	702	CATCH
```

Comment:
If we consider there solely the verbs then the mapping is fine but as a predicate 2638 FISHING would be a better match. On the other hand, following this logic, all the 'eat' instances above should be unmapped then.